docs/getting-started-guides/ubuntu/local.md

```diff
@@ -10,51 +10,24 @@ Running Kubernetes locally has obvious development advantages, such as lower cos
 
 The purpose of using [LXD](https://linuxcontainers.org/lxd/) on a local machine is to emulate the same deployment that a user would use in a cloud or bare metal. Each node is treated as a machine, with the same characteristics as production. Each node is a separate container, which runs Docker containers and `kubectl` inside (see [Cluster Intro](https://kubernetes.io/docs/tutorials/kubernetes-basics/cluster-intro/) for more info).
 
-{% capture prerequisites %}
```

Comment:
There needs to be a Prerequisites section, otherwise the page will display an error:

![screen shot 2017-07-05 at 1 16 05 pm](https://user-images.githubusercontent.com/24798125/27883092-2a58b2ce-6184-11e7-834d-3c8c23282d62.png)

You can see a preview build of the page here: https://deploy-preview-4278--kubernetes-io-vnext-staging.netlify.com/docs/getting-started-guides/ubuntu/local/